lib/lens/version.rb

```diff
@@ -1,3 +1,3 @@
 module Lens
-  VERSION = "0.0.8"
+  VERSION = "0.0.8.2"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.